src/MenuStyle.php

```diff
@@ -150,23 +160,25 @@ class MenuStyle
      * @var array
      */
     private static $defaultStyleValues = [
-        'fg' => 'white',
-        'bg' => 'blue',
-        'width' => 100,
-        'paddingTopBottom' => 1,
-        'paddingLeftRight' => 2,
-        'margin' => 2,
-        'selectedMarker' => '● ',
-        'unselectedMarker' => '○ ',
-        'itemExtra' => '✔',
-        'displaysExtra' => false,
-        'titleSeparator' => '=',
-        'borderTopWidth' => 0,
-        'borderRightWidth' => 0,
+        'fg'                => 'white',
+        'bg'                => 'blue',
+        'width'             => 100,
+        'paddingTopBottom'  => 1,
+        'paddingLeftRight'  => 2,
+        'margin'            => 2,
+        'selectedMarker'    => '● ',
+        'unselectedMarker'  => '○ ',
+        'toggledMarker'     => '[●] ',
```

Comment:
```suggestion
        'checkedMarker'     => '[●] ',
```

Reply:
Also I think I prefer the tick from your example as the default!

Reply:
`[✔]` or `✔`?

Reply:
`[✔]` please 👍